builtin/logical/pki/path_tidy.go

```diff
@@ -1381,6 +1500,50 @@ func (b *backend) doTidyCrossRevocationStore(ctx context.Context, req *logical.R
 	return nil
 }
 
+func (b *backend) doTidyAcme(ctx context.Context, req *logical.Request, logger hclog.Logger, config *tidyConfig) error {
+	b.acmeAccountLock.Lock()
+	defer b.acmeAccountLock.Unlock()
+
+	sc := b.makeStorageContext(ctx, req.Storage)
+	list, err := sc.Storage.List(ctx, acmeThumbprintPrefix)
+	if err != nil {
+		return err
+	}
+
+	b.tidyStatusLock.Lock()
+	b.tidyStatus.acmeAccountsCount = uint(len(list))
+	b.tidyStatusLock.Unlock()
+
+	baseUrl, _, err := getAcmeBaseUrl(sc, req.Path)
+	if err != nil {
+		return err
+	}
+
+	acmeCtx := &acmeContext{
+		baseUrl: baseUrl,
+		sc:      sc,
+	}
+
+	for _, thumbprint := range list {
+		b.tidyAcmeAccountByThumbprint(b.acmeState, acmeCtx, thumbprint, config.SafetyBuffer, config.AcmeAccountSafetyBuffer)
```

Comment:
It looks like this returns an error, should it be handled?

Reply:
I was thinking that an error on any particular account shouldn't stop the tidy, but I probably should log it, so thank you.  Logging added.